### PR TITLE
Manuscript workflows

### PR DESCRIPTION
**Pull request by Integral**
We want to use workflows inside a manuscript editor. This comes with a few examples: add new reference and editing refs using workflows.

Note: This is just the first iteration. We want to merge it and improve UX later.

### CODE REVIEW

File: src/article/editor/EditorPackage.js [hunk 1]
```diff
@@ -80,6 +80,8 @@ import {
 import InsertTableTool from './InsertTableTool'
 import SchemaAwareToggleListCommand from './SchemaAwareToggleListCommand'
 import ArticleNavPackage from '../ArticleNavPackage'
+// Workflows
+import AddReferenceWorkflow from '../metadata/AddReferenceWorkflow'
```
[the remark this answers is not in the document]
Comment (Integral) [reply src/article/editor/EditorPackage.js]: yes, i also thought about this...
do we want to introduce shared styles as well?

[the remark this answers is not in the document]
Comment (oliver----) [reply src/article/editor/EditorPackage.js]: Yes, I guess

[the remark this answers is not in the document]
Comment (oliver----) [reply src/article/editor/EditorPackage.js]: After my (hoperfully soon) merge, we should do a tidy-up. Eliminate dead code and reduce code/CSS redundancy

[the remark this answers is not in the document]
Comment (Integral) [reply src/article/editor/EditorPackage.js]: Nice, meanwhile I will remove dead code from RefListComponent.

[the remark this answers is not in the document]
Comment (oliver----) [reply src/article/editor/EditorPackage.js]: 👍

**Pull request merged**